test/parallel/test-cluster-inspector-debug-port.js

```diff
@@ -13,7 +13,7 @@ if (cluster.isMaster) {
 
   function fork(offset, execArgv) {
     if (execArgv)
-      cluster.setupMaster({execArgv});
+      cluster.setupMaster({args: execArgv});
```

Comment:
Okay figured out this one for the moment and fixed in latest commit. Now I need to figure out how to run code inside the worker process - which I'm not sure how to do since it breaks immediately with `--inspect-brk`, orphaning the child. Any ideas?